src/main/java/frc/robot/Constants.java

```diff
@@ -23,9 +23,12 @@ public static class FieldConstants {
     public static final double kFieldLength = 0.0;
     public static final double kFieldWidth = 0.0;
   }
-
+  
+  //TODO: Fill in the parameters
   public static class PhotonVisionConstants {
-    public static final Transform3d kCameraToRobotOffset = new Transform3d();
+    public static final String kCameraName1 = "OV9281";
```

Comment:
I wonder how camera naming is going to work when we have two OV9281 cameras connected at the same time.